### PR TITLE
Add details to OpenRefine setup as per Issue 234.md

### PR DESCRIPTION
**Pull request by stephanieuwa**
Adding details about the command line window when OpenRefine first runs as per Issue #234. As raised in the Issue, the Command Line Window was mentioned in the 'Exit' section, but not introduced at the start of the lesson.

### CODE REVIEW

**Review by ostephens — CHANGES_REQUESTED** [review]
Thanks for this @stephanieuwa - just a couple of minor issues before I think this is ready to merge

File: setup.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ If you are using an older version of OpenRefine, it is recommended you upgrade t
 
 Please follow OpenRefine's manual to [install](https://docs.openrefine.org/manual/installing) and [run](https://docs.openrefine.org/manual/running) it. 
 
+When running OpenRefine, initially a "Command Line Window" will open. This is a window with a black background. As OpenRefine runs, lines of text will appear in the command line window. Then the Open Refine interface will open in your default web browser. You do not need to interact with the command line window. Leave it open in the background, and work on datasets in your web browser.
```
[the remark this answers is not in the document]
Comment (stephanieuwa) [reply setup.md]: Thanks @ostephens, that's really helpful. I've made those edits. I'm still quite new to GitHub so I don't know if there's anything further I need to do to submit those additional edits to you? They appear to be part of this pull request but I'm not 100% certain. Thanks.

[the remark this answers is not in the document]
Comment (ostephens) [reply setup.md]: Thanks @stephanieuwa - that's exactly the right thing - as soon as you add further edits to the branch you used to create the pull request they appear on the pull request

**Comment by ostephens** [comment]
Thanks so much @stephanieuwa

**Pull request merged**